src/main/java/org/folio/inventory/parsing/ModsParser.java

```diff
@@ -36,7 +37,7 @@ public ModsParser(CharacterEncoding characterEncoding) {
     DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
     factory.setNamespaceAware(false);
     DocumentBuilder builder = factory.newDocumentBuilder();
-    Document doc = builder.parse(IOUtils.toInputStream(xml, "UTF-8"));
+    Document doc = builder.parse(new InputSource(new StringReader(xml)));
```

Comment:
we need to think about closing resources, class StringReader has close method, do we need to invoke it?

Reply:
![StringReader close javadoc](https://user-images.githubusercontent.com/88028273/131649717-0ba2a6cb-335f-4f8a-905c-101eaba301dd.png)
There's little value in closing a StringReader, since it doesn't hold an external resource (only Java-managed memory, not a file handle or native memory), so it's fine to omit it

Reply:
Thanks for checking